hoodie-client/src/test/java/com/uber/hoodie/common/HoodieTestDataGenerator.java

```diff
@@ -209,9 +215,27 @@ public static void createSavepointFile(String basePath, String commitTime) throw
     }
   }
 
+  public static void createCleanFiles(String basePath, String commitTime) throws IOException {
+        Path commitFile =
```

Comment:
But you have empty payload here anyways right.. This class was meant only for payload generation.. Please move this to HoodieTestUtils.